static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/SubscriptionDetails.tsx

```diff
@@ -183,16 +183,30 @@ export const SubscriptionDetails = forwardRef<HTMLDivElement, Props>(
               <>
                 {subscription.statuses.has_access_to_support &&
                 subscription.type !== UserSubscriptionType.Trial ? (
-                  <Button
-                    appearance="positive"
-                    className="p-subscriptions__details-action"
-                    data-test="support-button"
-                    disabled={editing}
-                    element="a"
-                    href="https://portal.support.canonical.com/"
-                  >
-                    Support portal
-                  </Button>
+                  <>
+                    <Button
+                      appearance="positive"
+                      className="p-subscriptions__details-action"
+                      data-test="support-button"
+                      disabled={editing}
+                      element="a"
+                      href="https://portal.support.canonical.com/"
+                    >
+                      Support portal
+                    </Button>
+                    <Button
```

Comment:
I wonder if we need a separate condition has this allows you to click the attach button on expired subscriptions.

Reply:
It is inside a condition that renders the button only on subscriptions that have access to support. I don't think expired subscriptions have access to support